pkg/util/constants.go

```diff
@@ -11,3 +11,34 @@ const (
 	DeckSSLEnabledProp      = "security.uiSecurity.ssl.enabled"
 	ClouddriverName         = "clouddriver"
 )
+
+var (
+	SpinnakerServices = map[string]struct{}{
+		"clouddriver": {},
+		"orca":        {},
+		"echo":        {},
+		"fiat":        {},
+		"igor":        {},
+		"rosco":       {},
+		"front50":     {},
+		"kayenta":     {},
+		"gate":        {},
+		"dinghy":      {},
+		"terraformer": {},
+	}
+	SpinnakerJavaServices = map[string]struct{}{
```

Comment:
I would suggest having a single map instead of a map per service type. Something like this:
```
var (
	SpinnakerServices = map[string]SpinSvcMetadata{
		"clouddriver": {
			Name:       "clouddriver",
			Type:        "java",
		},
         }
)

type SpinSvcMetadata struct {
	Name       string
	Type       string
}
```
Along with appropriate methods in this file to fecth all service names, all java services and so on.